apps/mull-ui/src/app/pages/register/register.scss

```diff
@@ -0,0 +1,65 @@
+@import 'variables';
+
+.register-mull-logo {
+  padding-top: 8em;
+  display: block;
+  margin: 0 auto;
+  width: 30%;
+  padding-bottom: 3em;
+}
+
+.register-container {
+  display: grid;
```

Comment:
Would it not be more simple to use a flexbox instead of a grid? I see you're centering things individually and a flexbox could help. Since every thing is in a column, I don't think grid is necessary.

Reply:
don't fear the flex @JimLau49

Reply:
Simply used grid because the gap property works on all browsers while the gap property on flex works only on some browsers

Reply:
I guess it's fine for now. If it causes problems in the future. I can convert it into a flexbox. 🚀